index.html

```diff
@@ -283,9 +283,9 @@ <h2 class="lang-h2">Skills</h2>
         <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
             please get in touch.</h2>
         <form class="form">
```

Comment:
- [ ] Kindly add the action and method attribute to the form with the action attribute's value having your [Form Spree](https://formspree.io/html/) URL you created on Form Spree and the method is set to POST that allows for posting form data to that Form Spree URL so messages can be sent to your email. Please work on that 👍 
- [ ] Kindly click on the link above, go to the Form Spree website, create your account, then create a form and take the URL of the form created and pass it to your form's action attribute. 👍